SortApp/SortApp.Presentation/MainWindow.xaml

```diff
@@ -5,8 +5,32 @@
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:SortApp.Presentation"
         mc:Ignorable="d"
-        Title="MainWindow" Height="350" Width="525">
+        Title="SortApp" Height="350" Width="525">
     <Grid>
-        
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"></RowDefinition>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="100"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*"></RowDefinition>
+        </Grid.RowDefinitions>
+        <Menu>
+            <MenuItem Header="File">
+                <MenuItem Header="Exit"/>
+            </MenuItem>
+        </Menu>
+        <Label x:Name="InputArrayLabel" Content="Input array" Grid.Row="1"/>
+        <TextBox x:Name="OriginalDataTextBox" Grid.Row="2" Margin="5" VerticalScrollBarVisibility="Auto" TextWrapping="Wrap" Text="{Binding Path=OriginalData, UpdateSourceTrigger=PropertyChanged}"/>

```

Comment:
This line seems too long. If it's longer that 120 symbols - break it into several